packages/features/eventtypes/components/EventTypeDescription.tsx

```diff
@@ -78,11 +81,21 @@ export const EventTypeDescription = ({
               </Badge>
             </li>
           )}
-          {eventType.metadata?.managedEventConfig && !isPublic && (
-            <Badge variant="gray" startIcon="lock">
-              {t("managed")}
-            </Badge>
-          )}
+          {eventType.metadata?.managedEventConfig &&
```

Comment:
@vijayraghav-io can we not have this much nesting? Reason I say is because if someone in the future wants to add more logic to this it will get very messy very soon. I was thinking we have two variables, one for `isEventTypeManaged` and `isOwnerPresent` (the variables names can be more descriptive) and then we display component based on that.
```
const isEventTypeManaged = eventType.metadata?.managedEventConfig && !isPublic
const isOwnerPresent = !!eventType.teamAdmin 

if (isEventTypeManaged && isOwnerPresent) <>Render icon with tooltip here</>
if (isEventTypeManaged && !isOwnerPresent) <>Render just the icon</>
```
What do you think about this?

Reply:
I feel instead of having multiple conditions, this is more readable.